plugin/storage/integration/elasticsearch_test.go

```diff
@@ -206,7 +205,6 @@ func (s *ESStorageIntegration) initSpanstore(t *testing.T, allTagsAsFields, arch
 		IndexPrefix:       indexPrefix,
 		MaxSpanAge:        maxSpanAge,
 		TagDotReplacement: tagKeyDeDotChar,
-		Archive:           archive,
```

Comment:
so where do you initialize ArchiveSpanReader now?

Reply:
Yes, I apologise for my absent-mindedness 😅. Made the changes now.

Reply:
Hi, I tried looking into it, but why is the ES expected trace and actual trace differing? Is it because of the way I have initialized the Archive Reader/Writer and there is a specific way to do so?